dev/cnf/ant/liberty-ghe.xml

```diff
@@ -135,18 +135,18 @@ ${commits.since.latest.green.build}
     </target>
     
     <target name="taggingChecks">
```

Comment:
Is this code even used with open liberty.  Can we delete the task complete?  Or maybe even delete the entire file?

Reply:
I'll try removing the cnf/ant directory in the next build.